packages/admin/admin/src/appHeader/dropdown/AppHeaderDropdown.tsx

```diff
@@ -46,7 +46,6 @@ export function AppHeaderDropdown(inProps: AppHeaderDropdownProps) {
         children,
         buttonChildren,
         dropdownArrow = (isOpen: boolean) => (isOpen ? <ChevronUp /> : <ChevronDown />),
-        popoverProps,
```

Comment:
Removing `popoverProps` here will include them in `restProps` and will therefore now be applied to the `Button`. 

If the prop should be removed, it needs to be removed from the props type as well and a changeset needs to be added for the breaking change, mentioning to use `slotProps` instead.

Reply:
8c7a021e9324207c4671cf8d054b611dc58527d8